sdks/java/extensions/sorter/src/test/java/org/apache/beam/sdk/extensions/sorter/BufferedExternalSorterTest.java

```diff
@@ -75,8 +78,8 @@ public void testFallback() throws Exception {
     ExternalSorter mockExternalSorter = mock(ExternalSorter.class);
     InMemorySorter mockInMemorySorter = mock(InMemorySorter.class);
     Constructor<BufferedExternalSorter> constructor =
-        BufferedExternalSorter.class
-            .getDeclaredConstructor(ExternalSorter.class, InMemorySorter.class);
+        BufferedExternalSorter.class.getDeclaredConstructor(
+            ExternalSorter.class, InMemorySorter.class);
```

Comment:
:( could have sworn I had fixed up this method too, but apparently not. Fixed now